app/database/PostgresConnection.scala

```diff
@@ -0,0 +1,39 @@
+package database
+import anorm.SqlParser.str
+import anorm.{RowParser, SQL, SqlParser, ~}
+import com.nashtech.delivery.v1.anorm.conversions.Standard.columnToJsObject
+import com.nashtech.delivery.v1.models.json.{jsonReadsDeliveryAddress, jsonReadsDeliveryContact}
+import com.nashtech.delivery.v1.models.{Address, Contact, Delivery}
+import play.api.db._
+import play.api.libs.json._
+
+import javax.inject.Inject
+
+class PostgresConnection @Inject()(db: Database) {
+  def getAll(sqlForD: String): Seq[Delivery] = {
+    db.withConnection { implicit connection =>
+      SQL(sqlForD).as(deliveryParser.*)
```

Comment:
Have you tried using Delivery.parser from the generated parser?

Reply:
No, I was not aware of it. I will check.